### ISSUE TITLE
Windows Defender Detected Release Zips as Trojan:Win32/CryptInject!ml

### ISSUE THREAD

**Issue by alliedcvil** [yamashi/CyberEngineTweaks#273]
Been using this since the initial launch of the tweaks. Thanks for all of the hard work; however, today after I ran the latest Windows Defender security definition updates it detected each of the Release zips in particular for this repo as containing Trojan:Win32/CryptInject!ml. 

https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan:Win32/CryptInject!ml

More than likely a false positive, but I removed them immediately.

**Comment by fathuraw** [comment]
same here, got alert from windows defender

**Comment by bryannolen** [comment]
If you got it from the releases page here, it is a false positive. If you got it anywhere else, you might have a problem.

https://www.virustotal.com/gui/file/d9aa90b200c633c330db1bad03098afaf202b128e502d2fa770e486423a03edb/detection

**Comment by LegendModzYT** [comment]
I would recommend not using Windows Defending seeing as it can have many false positives.

**Comment by bryannolen** [comment]
Defender has a relatively low false positive rate, especially if you keep it properly updated. In this case, the latest version of defender does not trigger this FP (as seen in the VT link I posted, and from my own testing.)

**Comment by helium-three** [comment]
Can confirm that Windows Defender security intelligence version 1.329.1017.0 (current as of 25-12-2020T0410Z) gets a hit for the above referenced trojan from the zip I downloaded on 19-12-2020T0012Z but _not_ any binaries later than that on my machine. It appears that the next version I downloaded was significantly smaller due to major code changes. Looks like a false positive but I have no idea what heuristic it's getting caught on. Manually scanning folder containing about a dozen releases does not hit any of the others for me.
